examples/controlnet/train_controlnet_sdxl.py

```diff
@@ -838,6 +838,7 @@ def main(args):
         vae_path,
         subfolder="vae" if args.pretrained_vae_model_name_or_path is None else None,
         revision=args.revision,
+        force_upcast=True if args.pretrained_vae_name_or_path is None else False,
```

Comment:
Works for me, but `True` is also the default so should be fine without no?

Reply:
This was needed before this commit got merged: https://huggingface.co/madebyollin/sdxl-vae-fp16-fix/commit/9292e146692bb47baa25d8d64e86e46b5f524101
Now this is not needed anymore, I agree